cmd/mattermost/commands/config.go

```diff
@@ -217,6 +229,40 @@ func configSetCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func configMigrateCmdF(command *cobra.Command, args []string) error {
+	// Parse source config; defaults to global --config unless overwritten by --from
+	from, err := command.Flags().GetString("from")
+	if err != nil {
+		return errors.Wrap(err, "failed reading source config parameter")
+	}
+	if from == "" {
+		from = viper.GetString("config")
+	}
+
+	// Parse destination config store - MarkFlagRequired handles errors here
+	to, err := command.Flags().GetString("to")
```

Comment:
nit: if you are discarding the error, probably you should be discarding the error, something like `to, _ := ...`